packages/common/src/authentication/AuthContext.tsx

```diff
@@ -175,8 +175,7 @@ export const AuthProvider: FC = ({ children }) => {
       username: mostRecentlyUsedCredentials?.username ?? '',
       store,
     });
-    const authCookie = getAuthCookie();
-    const newCookie = { ...authCookie, store };
+    const newCookie = { ...cookie, store };
```

Comment:
ah. yeah - the issue is that when you change stores it was reading from the cookie. If you have recently logged in, the cookie might not be set yet, and that would wipe out the token, cause an auth error and annoy Mark.